### PR TITLE
Feature/filter experiments

### PR DESCRIPTION
**Pull request by nibivi77**
# Description

[Filter for the experiments](https://trello.com/c/JEvAH1TO/168-filter-for-the-experiments)

**Description:** Give users an option to filter through experiment cards once there are many of them. 

**Done when:** We have the labels for the filters determined, can add states as we go

**Mock-ups**
image
image

**On the Branch**
image
image

## Acceptance Criteria

We can add options and re-style as we go. The functionality will remain.

## Test Instructions

1. navigate to branch name: Feature/FilterExperiments
2. git pull and npm run dev
3. navigate to experiments page and see the filter options.
4. Check different screen widths.

## Help Requested

- I need help with testing. I have no idea how to test this. I only added the a11y unit test.
- The same applies for storybook. I am not sure how to set it up properly.

## Checklist

- [x] Strings use placeholders for translation (No hard coded strings)
- [ ] Unit tests have been added/updated

## Product and Sprint Backlog

[Filter for the experiments](https://trello.com/c/JEvAH1TO/168-filter-for-the-experiments)

### CODE REVIEW

File: components/atoms/FilterExperiments.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        type="radio"
+        className="radioButton absolute -left-full -top-full"
+        name="radioButtonFilter"
+        defaultChecked={option.id === 0}
+        value={option.text}
+        id={option.id}
+        onChange={props.onChange}
+      />
+      <label
+        className={`float-left text-xs px-5 py-3 cursor-pointer border border-solid border-gray-600 border-opacity-50 mr-0 mb-6 ${
```
Comment (will0684) [inline components/atoms/FilterExperiments.js:21]: Don't forget to use "font-body" class to make sure your labels are using the "Noto Sans" font.

File: components/atoms/FilterExperiments.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <div
+      className="mt-4 md:flex md:mt-6"
+      role="radiogroup"
+      aria-labelledby="filterExperiments"
+      data-cy={props.dataCy}
+    >
+      <label
+        id="filterExperiments"
+        htmlFor="radioButtonGroup"
+        className="pb-3 pt-2 pr-4 text-sm md:text-base"
```
Comment (will0684) [inline components/atoms/FilterExperiments.js:44]: This should be "font-body" as well.

**Review by will0684 — CHANGES_REQUESTED** [review]
Great work here Ben, just some font issues that need fixing. Might also be worth considering implementing a cypress integration test for the Experiments page that tests the filter by clicking it and then checking the list on the screen for the right experiments in the list.

File: components/atoms/FilterExperiments.js (hunk 2 above)
Comment (nibivi77) [reply components/atoms/FilterExperiments.js]: It has been done.

File: components/atoms/FilterExperiments.js (hunk 1 above)
Comment (nibivi77) [reply components/atoms/FilterExperiments.js]: It has been done.

**Comment by Moro-Code** [comment]
Closing for now will reopen once completed. Code needs some work

**Pull request closed without merging**